cmd/exec/operations.go

```diff
@@ -527,16 +528,24 @@ func executeEvent(url string) (string, error) {
 		}
 
 		if resp.StatusCode != http.StatusOK {
-			log.Fatalf("Failed to send event: %v", err)
+			err = errors.New("Failed to send Event")
+			return "", err
 		}
 	}
 
-	_, ok := event["id"].(string)
-	if ok {
-		return event["id"].(string), nil
+	x, exist := event["id"]
+	if !exist {
+		err = errors.New("Event id not existing")
+		return "", err
+	}
+
+	id, ok := x.(string)
+	if !ok {
+		err = errors.New("Event id not existing")
```

Comment:
This error message is slightly misleading. It shows up when an ID was defined, it just didn't unmarshal as a string.